tools/topology/sof/pipe-processing-playback.m4

```diff
@@ -38,74 +21,26 @@ CONTROLBYTES_PRIV(PP_SETUP_CONFIG,
 `	0x01,0x00,0x00,0x00,0x04,0x00,0x00,0x00,'
 `	0x0C,0x00,0x00,0x00,0x02,0x00,0x00,0x00,'
 `	0x05,0x00,0x00,0x00,0x0C,0x00,0x00,0x00,'
-`	0x02,0x00,0x00,0x00"'
+`	0x02,0x00,0x00,0x00"''
 )
-
-# Post process Bytes control for setup config
-C_CONTROLBYTES(Post Process Setup Config, PIPELINE_ID,
-	CONTROLBYTES_OPS(bytes),
-	CONTROLBYTES_EXTOPS(void, 258, 258),
-	, , ,
-	CONTROLBYTES_MAX(void, 300),
-	,
-	PP_SETUP_CONFIG)
-
-
-# Post process runtime params
-CONTROLBYTES_PRIV(PP_RUNTIME_PARAMS,
-`	bytes "0x53,0x4f,0x46,0x00,0x01,0x00,0x00,0x00,'
-`	0x00,0x00,0x00,0x00,0x00,0x10,0x00,0x03,'
-`	0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,'
-`	0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00"'
+define(`CA_SETUP_CONTROLBYTES_MAX', 300)
+
+# codec Post Process runtime params
+define(`CA_RUNTIME_CONTROLBYTES',
+``       bytes "0x53,0x4f,0x46,0x00,0x01,0x00,0x00,0x00,'
+`       0x00,0x00,0x00,0x00,0x00,0x10,0x00,0x03,'
+`       0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,'
+`       0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00"''
 )
+define(`CA_RUNTIME_CONTROLBYTES_MAX', 157)
 
-# Post process Bytes control for runtime config
-C_CONTROLBYTES(Post Process Runtime Params, PIPELINE_ID,
-        CONTROLBYTES_OPS(bytes),
-        CONTROLBYTES_EXTOPS(void, 258, 258),
-        , , ,
-        CONTROLBYTES_MAX(void, 157),
-        ,
-        PP_RUNTIME_PARAMS)
-
-#
-# Components and Buffers
-#
+define(`CA_SCHEDULE_CORE', 1)
```

Comment:
Should we add pipeline ID to this?

Reply:
I don't think so, pipeline ID is brought by the upper layer.
I observed that in previous pipe-processing-playback.m4, the schedule core of codec adapter (PP_CORE) is set to 1 as default which is different than others (core=0).